go/vt/wrangler/materializer.go

```diff
@@ -767,8 +768,17 @@ func (wr *Wrangler) ExternalizeVindex(ctx context.Context, qualifiedVindexName s
 			}
 			state := row[1].ToString()
 			message := row[2].ToString()
-			if sourceVindex.Owner == "" {
-				// If there's no owner, all streams need to be running.
+			var bls binlogdatapb.BinlogSource
+			sourceBytes, err := row[3].ToBytes()
```

Comment:
sourceBytes looks unused.

Reply:
See line 776 🙂 We merged it already, but I appreciate you looking! ❤️